cypress/locators/help/locators.js

```diff
@@ -1,4 +1,3 @@
-// component preview locators
-const HELP_ICON_PREVIEW = 'a[data-component="help"]';
-
-export default HELP_ICON_PREVIEW;
+export const HELP_PREVIEW = '[data-component="help"]';
+export const HELP_ICON =
+  '[data-component="help"] > span[data-component="icon"]';
```

Comment:
what is the point of doing this ? why we can't use `icon()` function from `common-locators` ?